applovin_max/example/lib/main.dart

```diff
@@ -92,11 +92,17 @@ class _MyAppState extends State<MyApp> {
         // Interstitial ad failed to load
         // We recommend retrying with exponentially higher delays up to a maximum delay (in this case 64 seconds)
         _interstitialRetryAttempt = _interstitialRetryAttempt + 1;
+        if (_interstitialRetryAttempt > 6) {
```

Comment:
nit: I think we can extract the `6` into a constant, like `MAX_EXPONENTIAL_RETRY_COUNT` or something, so that we know whey we are failing after 6.

Reply:
I think this is a good idea. @alhiwatan - Could you create a docs ticket to update our docs accordingly for all platforms, and update RN and native demo apps?